gui/mozregui/addons_editor.py

```diff
@@ -17,7 +17,7 @@ def rowCount(self, index=QModelIndex()):
         return len(self.addons)
 
     def data(self, index, role=Qt.DisplayRole):
-        if role == Qt.DisplayRole:
+        if role in Qt.DisplayRole:
```

Comment:
just keep the == here! the in keyword is to check if something is in an iterable, e.g:

```
1 == 1  # True
1 in (1, 2, 3) # True
```